digits/templates/models/images/classification/new.html

```diff
@@ -366,6 +366,11 @@ <h4 style="display:inline-block;">Python Layers</h4>
                     {{form.use_mean.tooltip}}
                     {{form.use_mean(class='form-control')}}
                 </div>
+                <div class="form-group{{mark_errors([form.mean_path])}}">
```

Comment:
Support for client-side images could be implemented, but I guess you would then have to copy the specified image file to some directory on the server. Which directory should be used?